tests/integration-tests/tests/performance_tests/test_starccm.py

```diff
@@ -0,0 +1,95 @@
+import json
+import logging
+
+import boto3
+import pytest
+from assertpy import assert_that
+
+from remote_command_executor import RemoteCommandExecutionError, RemoteCommandExecutor
+
+# timeout in seconds
+STARCCM_INSTALLATION_TIMEOUT = 1800
+STARCCM_JOB_TIMEOUT = 600
+STARCCM_LICENCE_SECRET = "perf_integ_secret"
+TASK_VCPUS = 36  # vCPUs are cut in a half because multithreading is disabled
+BASELINE_ELAPSED_TIME = {8: 64.475, 16: 33.1723, 32: 17.8983}
+PERF_TEST_DIFFERENCE_TOLERANCE = 5
+
+
+def get_starccm_secrets(region_name):
+    secrets_manager_client = boto3.client("secretsmanager", region_name=region_name)
+    response = secrets_manager_client.get_secret_value(SecretId=STARCCM_LICENCE_SECRET)["SecretString"]
+    secrets = json.loads(response)
+    return secrets["podkey"], secrets["licpath"]
+
+
+def perf_test_difference(perf_test_result, number_of_nodes):
+    baseline_result = BASELINE_ELAPSED_TIME[number_of_nodes]
+    difference = perf_test_result-baseline_result
+    avg_result = (baseline_result+perf_test_result)/2
+    percentage_difference = abs(difference/avg_result)*100
+    percentage_difference *= -1 if difference < 0 else 1
+    return percentage_difference
+
+
+def starccm_installed(headnode):
+    cmd = "/shared/ec2-user/STAR-CCM+/16.02.008/STAR-CCM+16.02.008/star/bin/starccm+ --version"
```

Comment:
Improvement  for the next PR
The path of the shared directory should be the one injected in the cluster config for the property `SharedStorage[0].MountDir` so to avoid path redefinition and reduce error pronness.